File: Go/FizzBuzz/fizzbuzz.go
package main

import "fmt"

func main() {
	var fizz, buzz, fizzbuzz, maxCount int
	fizz, buzz, fizzbuzz, maxCount = 3, 5, 15, 101

	i := 0
	for i < maxCount {
		if i%fizzbuzz == 0 {
			fmt.Println("fizzbuzz")
		} else if i%fizz == 0 {
			fmt.Println("fizz")
		} else if i%buzz == 0 {
			fmt.Println("buzz")
		} else {
			fmt.Println(i)
		}

		i++
	}
}

File: Ruby/fizzbuzz/fizzbuzz.rb
for i in 0..100
  if i == 0
     puts i 
  elsif i % 15 == 0 then 
     puts "fizzbuzz" 
  elsif i % 3 == 0 then 
     puts "fizz" 
  elsif i % 5 == 0 then 
     puts "buzz" 
  else
     puts i 
  end
end
